### PR TITLE
Community badges soft launch

### PR DESCRIPTION
**Pull request by peteroneilljr**
Soft launch guidelines for community members to apply for the OPA community badges program.

### CODE REVIEW

File: COMMUNITY_BADGES.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+# OPA Community Badges
+
+Hello OPA Community Members, we thank you for taking the time to be a part of the community and for all of the work you’ve put in. As thanks, the OPA community now offers badges to recognize your contributions to the community.
```
Comment (anderseknert) [inline COMMUNITY_BADGES.md:3]: Suggest breaking this into two sentences:
```
Hello OPA Community Members! We thank you ...
```

File: COMMUNITY_BADGES.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+- Access to a private OPA Champions Slack Channel
+- Help from the OPA team to assist with community issues
+- Increased support for your OPA projects
+
+## Application Process
+
+To apply for any of the OPA badges, fill out the form at: https://airtable.com/shrrd45fRi7yO2k6i
+
+For any questions, please reach out to peteroneilljr@styra.com or @peteroneilljr in [Slack](https://slack.openpolicyagent.org/).
```
Comment (anderseknert) [inline COMMUNITY_BADGES.md:67]: Given how I think we can assume/require Slack membership, I think we can skip the email address here?

**Review by anderseknert — APPROVED** [review]
Looks good! Left two minor suggestions.

File: COMMUNITY_BADGES.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+Actively help and support the OPA community. Some places we will look for contributions are:
+
+- Slack
+- GitHub Discussions 
+- Stack Overflow
+- Meetups
+
+Being a champion in the OPA community can take many shapes and forms; as such, we do not want to limit this award to a static number of posts or comments. Instead, we are looking for members who take an active role in helping to build the community, or we look for members that want to have a more active role. 
+
+### Benefits
```
Comment (anderseknert) [inline COMMUNITY_BADGES.md:57]: Could we include some swag (t-shirt, stickers, etc) in this category?

**Review by srenatus — DISMISSED** [review]

File: COMMUNITY_BADGES.md (hunk 3 above)
Comment (peteroneilljr) [reply COMMUNITY_BADGES.md:57]: Most definitely, we can iron this out after the soft launch.

File: COMMUNITY_BADGES.md (hunk 2 above)
Comment (peteroneilljr) [reply COMMUNITY_BADGES.md:67]: Yea most likely not necessary, but I think it's fine to leave it.

File: COMMUNITY_BADGES.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+  
+### Requirements
+
+- Deploy or Support OPA in a production environment
+- Add your organization to the ADOPTERS.md file
+
+### Benefits
+
+- Access to a private OPA Super Users Slack Channel
+- Support writing an OPA CFP to grow your portfolio
```
Comment (ashutosh-narkar) [inline COMMUNITY_BADGES.md:40]: What kind of support are we talking about here ? For ex, reviewing CFPs by OPA maintainers ?

File: COMMUNITY_BADGES.md (hunk 1 above)
Comment (peteroneilljr) [reply COMMUNITY_BADGES.md]: Updated

File: COMMUNITY_BADGES.md (hunk 4 above)
Comment (peteroneilljr) [reply COMMUNITY_BADGES.md:40]: Anyone on the OPA team is fine, the goal here is to get more organizations to share their story.

**Review by tsandall — APPROVED** [review]
LGTM!

**Pull request merged**